pandas/tests/util/test_hashing.py

```diff
@@ -353,3 +353,22 @@ def test_hash_collisions():
 
     result = hash_array(np.asarray(hashes, dtype=object), "utf8")
     tm.assert_numpy_array_equal(result, np.concatenate([expected1, expected2], axis=0))
+
+
+def test_hash_with_tuple():
```

Comment:
move near the existing test for tuple hashing.

I would compare the results of the hashing rather than just running them.

Reply:
do we have some other way of calculating `expected` for these?  just use the values i get locally?

Reply:
yep exactly, they should hash exactly the same on all platforms / versions of python

Reply:
updated